system/Autoloader/FileLocator.php

```diff
@@ -237,7 +239,7 @@ protected function getNamespaces()
         foreach ($this->autoloader->getNamespace() as $prefix => $paths) {
             foreach ($paths as $path) {
                 if ($prefix === 'CodeIgniter') {
-                    $system = [
+                    $system[] = [
```

Comment:
This seems like a functionally-impactful change. Was this a bug?

Reply:
Ah, I see: this is to enable both directories to map to `\CodeIgniter\`. Could this cause other problems for existing code?

Reply:
No. The return value is originally an array list `[['prefix' => prefix, 'path' => path], ...]`.
Adding an array for `\CodeIgniter\` for test as a last element does no harm.